src/Spice86.Core/Emulator/VM/EmulationLoop.cs

```diff
@@ -103,8 +106,12 @@ public class EmulationLoop {
             if (_listensToBreakpoints) {
                 _machineBreakpoints.CheckBreakPoint();
             }
-            _cpu.ExecuteNextInstruction();
+            //_cpu.ExecuteNextInstruction();
+            _cfgCpu.ExecuteNext();
             _timer.Tick();
+            if (_cpuState.Cycles > 100000000) {
```

Comment:
I think this is to be removed.

Reply:
Removed it, was to do benchmarks with infinite loops and still have the code finish before the end of the universe